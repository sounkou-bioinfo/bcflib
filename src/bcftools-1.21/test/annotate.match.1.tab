#CHROM	POS	REF	ALT	SCORE	STR	INT	FLT
chr1	10	A	T	11	xx1	11	1.1
chr1	10	A	T	22	xx2	22	2.2
