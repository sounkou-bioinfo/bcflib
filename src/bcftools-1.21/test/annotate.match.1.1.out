##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##INFO=<ID=STR,Number=1,Type=String,Description="">
##INFO=<ID=INT,Number=1,Type=Integer,Description="">
##INFO=<ID=FLT,Number=1,Type=Float,Description="">
##INFO=<ID=SCORE,Number=1,Type=Integer,Description="">
##contig=<ID=chr1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	10	.	A	C	.	PASS	STR=xx2;INT=22;FLT=2.2;SCORE=22
